examples/example_host.cpp

```diff
@@ -25,8 +25,12 @@ struct account
 class ExampleHost : public evmc::Host
 {
     std::map<evmc::address, account> accounts;
+    evmc_tx_context tx_context{};
 
 public:
+    explicit ExampleHost() = default;
```

Comment:
The `explicit` is only needed for constructors with single argument.

Reply:
Fixed.